src/plugin/helpers/__tests__/fixtures.ts

```diff
@@ -2,13 +2,19 @@ import { Schema } from "prosemirror-model";
 import { schema as basicSchema } from "prosemirror-schema-basic";
 import type { NodeBuilder } from "prosemirror-test-builder";
 import { builders } from "prosemirror-test-builder";
+import { createNestedElementField } from "../../fieldViews/NestedElementFieldView";
 import { createRepeaterField } from "../../fieldViews/RepeaterFieldView";
 import { createTextField } from "../../fieldViews/TextFieldView";
 import { RepeaterFieldMapIDKey } from "../constants";
 import { createEditorWithElements, createNoopElement } from "../test";
 import { maxLength, required } from "../validation";
 
 export const elements = {
+  nestedElement: createNoopElement({
+    content: createTextField({
```

Comment:
Interested in this as it seems to be doing something similar but different to the nestedElementField below. Also it's called nestedElement, but seems to be a text no-op element with a text field?

Reply:
Unclear naming on my part I think - this is essentially a test element, separate from the 'example' element used in the tests, to put in the `nestedElementField` in the tests.

I think `exampleElementToNest` might be better